src/python/grpcio/grpc/__init__.py

```diff
@@ -753,6 +753,41 @@ def peer(self):
         raise NotImplementedError()
 
     @abc.abstractmethod
+    def peer_identity(self):
```

Comment:
Shouldn't a plural name (`peer_identities`) be used for something that can be one, but can also be many?

What's the cardinality of this anyway? Can it be zero-or-more (as well as `None`) or is it always one-or-more (or `None`)?